evennia/settings_default.py

```diff
@@ -125,6 +125,10 @@
 # file sizes down. Turn off to get ever growing log files and never
 # loose log info.
 CYCLE_LOGFILES = True
+# Number of lines to append to rotating channel logs when they rotate
+NUM_LOG_TAIL_LINES = 20
```

Comment:
This is only for channel logs, so that should be reflected in the names, such as `CHANNEL_LOG_FILE_TAILNUM` and `CHANNEL_LOG_MAXSIZE`.